src/convert.c

```diff
@@ -180,6 +180,12 @@ int convertToCandidate(struct raft *r, bool disrupt_leader)
     return 0;
 }
 
+void leader_noop_barrier_cb(struct raft_barrier *req, int status)
```

Comment:
Nit: I'd name this `convertInitialBarrierCb` or something like that, for consistency with the naming convention (camelCase, and private function names start with the module name).

Reply:
hehe, I just keep sticking to the underscores, will try to remember next time :-)